app/Services/CampaignService.php

```diff
@@ -220,6 +220,7 @@ public function getPendingPostTotals($campaigns)
                 ->leftJoin('posts', 'signups.id', '=', 'posts.signup_id')
                 ->selectRaw('signups.campaign_id, count(posts.id) as pending_count')
                 ->where('status', '=', 'pending')
+                ->where('posts.deleted_at', '=', null)
```

Comment:
Ahh, good catch! I wonder if we could re-write this query to use the Eloquent builder (which would automatically take this into account), but I could see that getting tricky perhaps.